flake.lock

```diff
@@ -385,11 +385,11 @@
         "stackage": "stackage"
       },
       "locked": {
-        "lastModified": 1641864086,
-        "narHash": "sha256-T/MUurQIxV4N015sU8M92+XVRLX+1LE6S02xL/X4ywM=",
+        "lastModified": 1645593720,
+        "narHash": "sha256-+5eoi7O5x0tk1H2mEioEUfad3YSGAwTH2M95vr2mJ+E=",
         "owner": "input-output-hk",
         "repo": "haskell.nix",
-        "rev": "f05b4ce7337cfa833a377a4f7a889cdbc6581103",
+        "rev": "a4e85e4714a8f11dbb36923bc63c5e93fa25032a",
```

Comment:
This is the commit _before_ your fix was introduced.

Reply:
So I've update the flake.lock, with `nix flake update`. I don't know why this isn't working for you locally :(

Reply:
Hmm. There still seems to be issues, like you mentioned with emanote. I'll have to have another look tomorrow.

Reply:
Maybe try #crossteam-ci too